include/SFML/System/Time.hpp

```diff
@@ -1,7 +1,7 @@
 ////////////////////////////////////////////////////////////
 //
 // SFML - Simple and Fast Multimedia Library
-// Copyright (C) 2007-2021 Laurent Gomila (laurent@sfml-dev.org)
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
```

Comment:
Did you mean to change the copyright year on a file you didn't modify?

Reply:
Actually Time files were changed too, then changed again and there is some leftover after those changes. I think I should revert all changes then.